rtfMRI/Messaging.py

```diff
@@ -0,0 +1,141 @@
+"""
+Messaging - Module to handle messaging between the client and server
```

Comment:
It looks like you have already put it quite some work in the messaging implementation. However, considering we do not have special requirements, I think it is worth evaluating any of the existing solutions, such as RabbitMQ, which @danielsuo has recently used for the rtFIN prototype.
https://github.com/brainiak/brainiak-cloud/blob/master/brainiak/cloud/publisher.py#L13

Reply:
Yes, I figured we would probably change the messaging layer in the future. The reason I did this way to start is I wanted to keep the deployment very simple and not require a third process as intermediary. So running local is trivial, no extra setup, and initial remote in the cloud same.